amivapi/settings.py

```diff
@@ -127,6 +127,32 @@
     'Best Regards,\nAMIV!'
 )
 
+# Email sent to people who have a new entry on the blacklists
+BLACKLIST_ADDED_EMAIL_W_PRICE = (
+    "You are BLACKLISTED!\n\nReason: %(reason)s\n\nThis means that you cannot "
```

Comment:
I'm not the biggest fan of all caps, but that's just my personal opinion and you can leave it if you want, but I would turn the intro into a full sentence and at the newlines before the reason, e.g. `You have been blacklisted for the following reason:\n\n%(reason)s`.

Before:
```
You are BLACKLISTED!

Reason: You have signed up for the coding weekend, but did not participate or pay.
```

After:
```
You have been blacklisted for the following reason:

You have signed up for the coding weekend, but did not participate or pay
```

I think this would make the reason stand out more.

Reply:
(If you disagree, thats fine. This change is not necessary)